imports/ui/lists/recentAgents.coffee

```diff
@@ -26,9 +27,20 @@ Template.recentAgents.onCreated ->
           if binding.days.value > 30
             binding.dm = true
         @recentAgents.insert(binding)
+        if @recentAgents.find(articleId: articleId).count() is 6
```

Comment:
It would be more efficient to put this in a separate loop over the articles so it only runs once per article instead of once per agent.